tests/pl/_fixtures.yaml

```diff
@@ -495,118 +495,118 @@ entities:
       device_class: window
 
   # https://www.home-assistant.io/integrations/sensor/
-  - name: "Moc pozorna urządzenia"
+  - name: "Moc pozorn[a|ą] urządzenia"
     id: "sensor.appliance_apparent_power"
     state: "2"
     attributes:
       device_class: apparent_power
       unit_of_measurement: "VA"
 
-  - name: "Czujnik powietrza w kuchni"
+  - name: "Czujnik[u|a] powietrza w kuchni"
     id: "sensor.kitchen_aqi_sensor"
     state: "50"
     attributes:
       device_class: aqi
 
-  - name: "Czujnik ciśnienia"
+  - name: "Czujnik[u] ciśnienia zewnętrznego"
```

Comment:
Shouldn't we include the `|a`? Seems identical to one above it. Can be usefull fome some tests.

Reply:
This is not necessary now. I start working with sensor.. if the new name will be needed then I will add it

Reply:
This name is just for running the test, and we don't have any tests in this PR at the moment 😀